### ISSUE TITLE
Question about training a new model with custom dataset

### ISSUE THREAD

**Issue by antonio2600** [open-mmlab/mmaction2#2144]
Hello, I have a question about training a new model with a custom dataset. What I have done is take a dataset and augment it with data augmentation so that it is balanced. Throughout the training it seems that he obtains good results almost at all epochs, as can be seen in the image. It is a pre-trained model based on another one that I trained from scratch. However, when the training ends and the model is validated, it has a very low hit from the top 1. Do you know what this is due to and how I can solve it?
Thanks.

![Captura desde 2022-12-19 19-20-44](https://user-images.githubusercontent.com/42941507/208493487-82579afe-daf5-4336-a7f5-c04f25f02056.png)

**Comment by antonio2600** [comment]
The image is this, sorry.

![Captura desde 2022-12-19 19-27-35](https://user-images.githubusercontent.com/42941507/208494642-f4f6dd0b-3031-4b43-90ad-0700bff79cfa.png)

**Comment by hukkai** [comment]
@antonio2600 did you mean your validation accuracy is good during training, but after training it has a very low hit from the top 1? Thanks

**Comment by antonio2600** [comment]
Yes, during training, the accuracy from the top 1 is good in every epochs, 0.9, 0.8, 0.7, 1. However in the final evaluation, the final accuracy from the top 1 is 0.57. I don't understand it.

**Comment by antonio2600** [comment]
These results, as you can see in the photo, I get during the training phase, as the epochs go by I get better results, until I reach the end of the training, which is what appears in the screenshot. However, then the validation accuracy gives me 0.57.

**Comment by hukkai** [comment]
These results (0.9, 0.8, 0.7, 1) in the photo are just training accuracies, which means that your model overfits.

**Comment by antonio2600** [comment]
And how can I do my model doesn't overfit?
